tee-launcher/launcher-docker-compose.yaml

```diff
@@ -2,16 +2,27 @@ version: '3.8'
 
 services:
   web:
-    image: barakeinavhnear/launcher:latest # todo (security) use spesific image hash  
+    image: barakeinavhnear/launcher:latest  # TODO (security): Replace with a specific image digest
```

Comment:
One issue is the missing image digest, and the other is that the Docker image is hosted in your personal DockerHub account. Could you please reference both related GitHub issue numbers in this TODO comment? It’ll make tracking easier.